configure

```diff
@@ -177,6 +177,7 @@ check_command() {
 
 check_command make
 check_command ${CC}
+check_command pkg-config
```

Comment:
I don't think this change belongs to the PR.  Let's move it to #1605.

Reply:
Ok, removed.